test/EndToEndTests/Tests/Client/Build.Desktop/Microsoft.Test.OData.Tests.Client.NetCore.csproj

```diff
@@ -1,147 +1,147 @@
+    <Compile Include="..\..\..\Services\TestServices\T4Clients\OperationService.cs" Link="OperationTests\OperationService.cs" />
+    <Compile Include="..\..\..\Services\TestServices\TestServiceUtil.cs" Link="ServiceWrapper\TestServiceUtil.cs" />
+    <Compile Include="..\AsynchronousTests\AsynchronousAttribute.cs" Link="AsynchronousAttribute.cs" />
+    <Compile Include="..\AsynchronousTests\AsynchronousDelayQueryTests.cs" Link="AsynchronousDelayQueryTests.cs" />
+    <Compile Include="..\AsynchronousTests\AsynchronousQueryTests.cs" Link="AsynchronousQueryTests.cs" />
+    <Compile Include="..\AsynchronousTests\AsynchronousSingletonClientTest.cs" Link="AsynchronousSingletonClientTest.cs" />
+    <Compile Include="..\AsynchronousTests\AsynchronousTestsExtensionMethodsDesktop.cs" Link="References\AsynchronousTestsExtensionMethodsDesktop.cs" />
+    <Compile Include="..\AsynchronousTests\AsynchronousUpdateTests.cs" Link="AsynchronousUpdateTests.cs" />
+    <Compile Include="..\AsynchronousTests\AsyncMethodTests.cs" Link="AsyncMethodTests.cs" />
+    <Compile Include="..\EndToEndTestBase.cs" Link="EndToEndTestBase.cs" />
+    <Compile Include="..\ServiceGeneratorFactory.cs" Link="ServiceGeneratorFactory.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Portable.Compatibility" Version="1.0.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.1.18" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.1.18" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.4.0" />
+    <PackageReference Include="System.Composition" Version="1.1.0" />
+    <PackageReference Include="System.Runtime.Serialization.Formatters" Version="4.3.0" />
+    <PackageReference Include="System.Runtime.Serialization.Primitives" Version="4.3.0" />
+    <PackageReference Include="System.ServiceModel.Primitives" Version="4.4.0" />
+    <PackageReference Include="xunit" Version="2.2.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\..\src\Microsoft.OData.Client\Microsoft.OData.Client.NetStandard.VS2017.csproj" />
+    <ProjectReference Include="..\..\..\..\..\src\Microsoft.OData.Core\Microsoft.OData.Core.NetStandard.VS2017.csproj" />
+    <ProjectReference Include="..\..\..\..\..\src\Microsoft.OData.Edm\Microsoft.OData.Edm.NetStandard.VS2017.csproj" />
+    <ProjectReference Include="..\..\..\..\..\src\Microsoft.Spatial\Microsoft.Spatial.NetStandard.VS2017.csproj" />
+    <ProjectReference Include="..\..\..\..\Common\Microsoft.Test.OData.DependencyInjection\Microsoft.Test.OData.DependencyInjection.NetCore.csproj" />
+  </ItemGroup>
+
   <ItemGroup>
     <Folder Include="References\" />
   </ItemGroup>
-
-</Project>
```

Comment:
Does this have any change? If it is a change that you did not intend then please revert this.